### PR TITLE
feat(mobile-download): Add mobile app store download links

### PR DESCRIPTION
**Pull request by jaredvu**
Screen Shot 2024-06-24 at 2 43 08 PM

Add download buttons to `DesktopHeader` and `AccountMenu`

---

## Views

* New: `<MobileDownloadLinks>`
  * Add new view to render links into `<HeaderDesktop>` and `<AccountMenu>`

* `<HeaderDesktop>` / `<AccountMenu>`
  * Add `MobileDownloadLinks`

## Components

* `<Icon>`
  * Add new Google Play Icon and add existing Apple Icon

* `<DropdownMenu>`
  * add `slotBottomContent`

## Packages

* `@dydxprotocol/v4-localization`
  * updated: v1.1.134 -> v1.1.138

---

### CODE REVIEW

File: src/views/MobileDownloadLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+import { IconName } from '@/components/Icon';
+import { IconButton } from '@/components/IconButton';
+import { Link } from '@/components/Link';
+import { VerticalSeparator } from '@/components/Separator';
+
+export const MobileDownloadLinks = ({ withBadges }: { withBadges?: boolean }) => {
+  const stringGetter = useStringGetter();
+
+  const androidUrl = document
```
Comment (rosepuppy) [inline src/views/MobileDownloadLinks.tsx:16]: nit: I wonder if we can leverage `getMobileAppUrl` from https://github.com/dydxprotocol/v4-web/blob/10d540188136caeeb0d9978d548d0fc7daa84015/src/views/dialogs/MobileDownloadDialog.tsx

**Review by rosepuppy — APPROVED** [review]
Code LGTM. The preview links don't have mobile urls injected, so need to test locally

File: src/views/MobileDownloadLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    .querySelector('meta[name="smartbanner:button-url-google"]')
```
Comment (tyleroooo) [inline src/views/MobileDownloadLinks.tsx:17]: I don't love running these selectors on every render. Could we just throw them in a useMemo to run on mount?

**Review by tyleroooo — COMMENTED** [review]
Rest looks good

File: src/views/MobileDownloadLinks.tsx (hunk 2 above)
Comment (jaredvu) [reply src/views/MobileDownloadLinks.tsx]: consolidated!

File: src/views/MobileDownloadLinks.tsx (hunk 1 above)
Comment (jaredvu) [reply src/views/MobileDownloadLinks.tsx]: Nice! Moved selector logic to a hook and used on `AccountMenu`, `MobileDownloadDialog` and `MobileDownloadLinks`

**Review by tyleroooo — APPROVED** [review]

**Pull request merged**